PD-11 Create Time Sync Article (#1706)
truenas/documentation: content/SCALE/SCALEUIReference/SCALEDashboard.md

---
title: Main Dashboard

> description: "This article provides information on the main Dashboard information cards (widgets) and includes basic instructions on synchronizing  system server and SCALE time, and customizing the display by moving, adding, or removing the widgets."

Reviewer: ```suggestion
description: "This article provides information on the main Dashboard information cards (widgets) and includes basic instructions on synchronizing system server and SCALE time, and customizing the display by moving, adding, or removing the widgets."
```

Click on the <span class="iconify" data-icon="ic:baseline-assessment"></span> icon to display the report screen that corresponds to that widget. For example, clicking the <span class="material-icons-outlined">assessment</span> icon on the **CPU** widget opens the **Reports > CPU** screen. 

### System Information Widget

> The **System Information** widget displays general information on the SCALE system, and includes a option to syncronize the system server time with TrueNAS SCALE time if they get out of sync.   

Reviewer: ```suggestion
The **System Information** widget displays general information on the SCALE system. It includes an option to synchronize the system server time with TrueNAS SCALE time if they get out of sync.   
```

Author: You know, I spelled it that way and darned if Hemingway didn't tell me I'd misspelled it!
